deep_groove_project/src/classification_dataset1/parameter_study.py

```diff
@@ -0,0 +1,94 @@
+from cls_set1 import cls_set1
+from cls_set1 import show_resultes
+import matplotlib.pyplot as plt
+import time
+
+
+def main():
+
+    params = {'batch_size': 20,
+              'shuffle': True,
+              'learning_rate': 1e-7,
+              'epochs': 200}
+
+    parameter_batch_size(params, output={"cli": False, "plot": False})
+    parameter_learning_rate(params, output={"cli": False, "plot": False})
+    parameter_epochs(params, output={"cli": False, "plot": False})
+
+
+def parameter_epochs(params, output):
+    st = time.time()
+    values = []
+    epochs = [10, 50, 100, 200, 500, 1000]
+    for i in range(len(epochs)):
+        params["epochs"] = epochs[i]
+        values.append(cls_set1(params, output=output))
+        values[len(values) - 1]['epochs'] = epochs[i]
+
+    elapsed_time = time.time() - st
+    print('Execution time:', elapsed_time, 'seconds')
+
+    for val, i in zip(values, range(len(values))):
```

Comment:
You do not need to comment on each line of code, but I can see that you thinking and coding in blocks. May you add comments per block? This would increase the understanding of coding story :)

Reply:
I added the docstings and comments to the blocks.

I hope they are enough. Furthermore, I will try to clean up the code in the following days to minimize my repeating code snippets. The comments helped to find these snippets.

Reply:
I simplified the show plot code by putting it into its own function